api/src/main/java/org/apache/iceberg/expressions/Expressions.java

```diff
@@ -301,4 +303,12 @@ public static <T> NamedReference<T> ref(String name) {
   public static <T> UnboundTerm<T> transform(String name, Transform<?, T> transform) {
     return new UnboundTransform<>(ref(name), transform);
   }
+
+  public static Term zorder(String... names) {
```

Comment:
If we move the `Zorder` class to core, these factory methods won't be possible anymore. I think that's okay, we can have Spark instantiate Zorder directly instead of calling these.

Reply:
done